app/mixins/editor-base-controller.js

```diff
@@ -41,7 +41,7 @@ export default Mixin.create({
     notifications: injectService(),
     clock: injectService(),
     slugGenerator: injectService(),
-
+    wordCount: 0,
```

Comment:
The actual logic is in the editor, the problem is that the wordcount itself should be outside of the editor element as we may want to place it elsewhere in the UI on smaller displays, etc.